### PR TITLE
fixed missing  text

### PR DESCRIPTION
**Pull request by RAJU2529**
as per the user report #8370 , so i added the word  **Password**

### CODE REVIEW

File: windows/security/threat-protection/security-policy-settings/minimum-password-length.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Types of password attacks include dictionary attacks (which attempt to use commo
 
 ### Countermeasure
 
-Configure the **** policy setting to a value of 8 or more. If the number of characters is set to 0, no password will be required.
+Configure the **Password** policy setting to a value of 8 or more. If the number of characters is set to 0, no password will be required.
```
Comment (illfated) [inline windows/security/threat-protection/security-policy-settings/minimum-password-length.md:79]: Thank you for the suggestion. I think this is a missing reference to the page topic, which is "Minimum password length", and that the text should reflect this:
```suggestion
Configure the **Minimum password length** policy setting to a value of 8 or more. If the number of characters is set to 0, no password will be required.
```

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**